robber/matchers/called.py

```diff
@@ -0,0 +1,42 @@
+from robber import expect
+from robber.matchers.base import Base
+
+
+class Called(Base):
+    """
+    expect(function).to.be.called()
+    """
+
+    def matches(self):
+        return Helper.is_mock(self.actual) and self.actual.called
+
+    def failure_message(self):
+        if not Helper.is_mock(self.actual):
+            return '{object} is not of type mock'.format(object=self.actual)
```

Comment:
Not sure if `object` here have any side-effects but we should not use this name anyway, it's a python keyword.